### DISCUSSION TITLE
How tweak Large veins? 1.18.2-3.3.0

### DISCUSSION THREAD

**Discussion by reddler97** [EwyBoy/OreTweaker#88] [Help Wanted!] [answered]
i am trying to disable all the ores. i am using the "remove_ores.json" Template, however when i use worldStripper i still find veins of copper and iron. a little research reveals these are Large veins. i have looked through other templates/defaults and have not found reference to these large veins. issue https://github.com/EwyBoy/OreTweaker/issues/58#issuecomment-1101652913 ends in saying that it is fixed with version 3.3.0 i have confirmed that i am using this latest version. a little help and/or examples of tweaking/disabling these would be helpful
![large copper vein](https://user-images.githubusercontent.com/8678826/209261103-6d3e38ec-0475-45bd-9d6e-d3237bde1968.png)

**Comment by EwyBoy** [comment, marked as answer]
Open the config file in the config folder. `OreTweaker.cfg` Inside it you can disable the large veins 🫠|

**Reply by UberVoyager** [reply]
I've also disabled large ore veins with the latest version on 1.18.2 and it worked for me. Do you have any other ore-changing mods active that might interfere with the generation?

**Reply by reddler97** [reply]
Well, that answers that. I can't believe my mind skipped over that when reading it and never returned to it. Still, for other use cases (not mine), creating a "shape" from those vanilla veins would be interesting.

**Reply by EwyBoy** [reply]
I have been looking into a method to spawn these types of ore veins.
The problem is that Mojang have hardcoded these veins straight into the terrain / chunk generator instead of spawning em like normal ores.
It makes it really hard to work with. Its not coded like anything else that spawns into the world like ores, structures, trees, lakes etc.
